### DISCUSSION TITLE
changing verbiage of TrashOperation?

### DISCUSSION THREAD

**Discussion by prescriptionlifeline** [Laravel-Backpack/community-forum#1134] [Q&A (Help)] [answered]
I was looking at https://backpackforlaravel.com/articles/tutorials/how-to-create-a-trash-deleted-section-in-backpack-crud and altho that looks pretty cool I don't like the verbiage "trashed". I mean, from the DB / PHP perspective, I get it, but the I'd rather people "archive" users vs "trash" them and I'd like it if the filter at the top said "Archived" vs "Trashed". And instead of "Destroy", when the Trashed / Archived filter is in use, you'd "Delete" users.

My question is...  how would I go about changing the verbiage?

**Comment by pxpm** [comment]
Hey @prescriptionlifeline 

You can change the relevant strings, and use the wording that's more adequate to your application by changing the language files.

Some of the strings are: 
```php
'crud::buttons.trash'
'crud::buttons.restore'
'crud::buttons.destroy'
// ...
```
You can find all the relevant strings in the source code, if you found anything that can't be changed in the language files, please let us know 🙏 

Cheers

**Comment by prescriptionlifeline** [comment, marked as answer]
I had to copy `vendor/backpack/pro/resources/lang/en` to `resources/lang/vendor/backpack/pro/en/trash.php` for this to work but, once I did, I was able to change the strings in that file.
